website/docs/docs/on-premises/scim.md

```diff
@@ -0,0 +1,284 @@
+
+For instance, if your Pactflow instance is running as `https://pactflow.mycompany.com` then you need to the set the
+`PACTFLOW_URL` environment variable to that address.
+
+### Next
+
+You can now test the running SCIM API by using a tool like `curl`. First, login to your Pactflow instance and create [a
+system account](/docs/user-interface/settings/users#system-accounts) and then copy the system account API token.
+
+For instance, assuming your Pactflow instance is running as `https://pactflow.mycompany.com` and your SCIM API is
+running as `https://pactflow-scim.mycompany.com` with its `PACTFLOW_URL` environment variable set to `https://pactflow.mycompany.com` and
+you have copied the system account API token, you can then run `curl -H 'Authorization: Bearer <PASTE THE SERVICE TOKEN HERE>' https://pactflow-scim.mycompany.com/Users`
+to fetch the users as a SCIM request.
+
+## Docker Compose example
+
+Here is an example docker compose setup (modified from the [Pactflow Docker Compose example](/docs/on-premises/docker-compose-example))
+to show how the SCIM API can be setup alongside the Pactflow instance. Run through the [Pactflow Docker Compose example](/docs/on-premises/docker-compose-example)
+first, and when that is working you can use this docker compose file to add SCIM as a façade.
+
+### 1. Authenticating to Quay.io
+
+Authenticate to Quay.io so that the docker engine is able to fetch our images.
+
+```
+docker login -u="<username>" -p="<password>" quay.io
+docker pull quay.io/pactflow/scim-api
+```
+
+### 2. Startup Pactflow with SCIM API
+
+Save the below file as `docker-compose.yml` into a temporary directory and then run `docker-compose up`. Make sure you
+have setup the Pactflow license file correctly as per [2. Pactflow license file](/docs/on-premises/docker-compose-example#2-pactflow-license-file).
+
+```yaml
+version: "3"
+
+networks:
+  frontend:
+    ipam:
+      config:
+        - subnet: 172.30.0.0/24
+
+services:
+  simplesaml:
+    image: kristophjunge/test-saml-idp
+    networks:
+      frontend:
+        ipv4_address: 172.30.0.2
+    logging:
+      driver: none # comment out the logging config to see the SAML server logs
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+    environment:
+      - SIMPLESAMLPHP_SP_ENTITY_ID=https://pactflow.io
+      - SIMPLESAMLPHP_SP_ASSERTION_CONSUMER_SERVICE=http://172.30.0.1/auth/saml/callback
+
+  pactflow:
+    image: quay.io/pactflow/enterprise
+    networks:
+      frontend:
+        ipv4_address: 172.30.0.3
+    depends_on:
+      - postgres
+    environment:
+      - PACTFLOW_HTTP_PORT=9292
+      - PACTFLOW_BASE_URL=http://172.30.0.1
+      - PACTFLOW_DATABASE_URL=postgres://postgres:password@postgres/postgres
+      # insecure settings only for the purposes of this demo! Not to be used in production.
+      - PACTFLOW_DATABASE_SSLMODE=disable
+      - PACTFLOW_REQUIRE_HTTPS=false
+      - PACTFLOW_SECURE_COOKIES=false
```

Comment:
PACTFLOW_SECURE_COOKIES not needed any more - it uses PACTFLOW_REQUIRE_HTTPS for both.

Reply:
This is copied from the other example, so I'll remove it from both